src/main/java/org/fogbowcloud/manager/core/models/orders/instances/ComputeOrderInstance.java

```diff
@@ -93,10 +93,10 @@ public void setSshExtraPorts(String sshExtraPorts) {
 	
 	public void setExternalServiceAddresses(Map<String, String> serviceAddresses) {
 		if (serviceAddresses != null) {
-			this.sshPublicAddress = serviceAddresses.get(CommonConfigurationConstants.SSH_SERVICE_NAME);
+            this.sshPublicAddress = serviceAddresses.get(CommonConfigurationConstants.SSH_SERVICE_NAME);
```

Comment:
Please, be careful with the Intelijj indentation.

Reply:
Okay!